src/log.h

```diff
@@ -141,20 +146,21 @@ uint32_t ly_log_options(uint32_t opts);
  */
 
 #define LY_LDGDICT  0x01 /**< Dictionary additions and deletions. */
-#define LY_LDGYANG  0x02 /**< YANG parser messages. */
-#define LY_LDGYIN   0x04 /**< YIN parser messages. */
 #define LY_LDGXPATH 0x08 /**< XPath parsing end evaluation. */
```

Comment:
I think it should be ordered, no reason to skip the values (except for backwards compatibility maybe, but one definitely needs to recompile an app when migrating from libyang 1 to 2).

Reply:
fixed